appinventor/docs/reference/other/manyscreens.html

```diff
@@ -435,15 +435,27 @@
               <section id="main-content" role="main">
                 <div id="content">  <div class="region region-content">
 
+
+       <!-- Here's where we insert the link to the Google Doc that holds the page contents, so people
+            can comment and edit, together with an iframe that holds the ame doc.  Apparently the
+            iFrame itself is not editable, which is why we link to the Doc twice. -->
+
+       <a href="https://docs.google.com/document/d/15W4-y0RMmXIpCQFQtaLXyOyBM9eDO4ULEWME9Sj0Vyo/edit#heading=h.3jgqgjwwqxts"
+	   target="_blank">
+           Click here for a version of this page on which you can comment.</a>
+
        <iframe width="100%" height="800"
-       src="https://docs.google.com/document/d/15W4-y0RMmXIpCQFQtaLXyOyBM9eDO4ULEWME9Sj0Vyo/pub?embedded=true"></iframe>
-                    <!-- Content -->
+	       src="https://docs.google.com/document/d/15W4-y0RMmXIpCQFQtaLXyOyBM9eDO4ULEWME9Sj0Vyo/edit#heading=h.3jgqgjwwqxts">
```

Comment:
@halatmit I recommend strongly against using an editable version in the iframe. There is a fixed (small) limit to the number of people who can visit a document in edit mode. I would suggest going back to the published HTML version inside the iframe and keeping the editable version above it for people who want to comment.